click/_bashcomplete.py

```diff
@@ -68,7 +77,12 @@ def do_complete(cli, prog_name):
     except IndexError:
         incomplete = ''
 
-    for item in get_choices(cli, prog_name, args, incomplete):
+    try:
+        last_complete = cwords[cword - 1]
```

Comment:
@untitaker yeah... it would look like `args[0]` would be the same as `cwords[cword - 1]`

I got confused because within `get_choices` it wasn't, but was because I was checking after the line `ctx = resolve_ctx(cli, prog_name, args)`.
It seems that line actually modifies args!

I'll fix this up now and re-submit